tests/src/lib.rs

```diff
@@ -61,14 +60,41 @@ mod tests {
 	}
 
 	#[test]
-	#[ignore]
 	fn test_admin_release() {
+		let config: VestingConfig = Default::default();
 		let mut vesting = VestingContract::deployed();
 		vesting.pause(Sender(ADMIN));
+		vesting.set_block_time(config.admin_release_duration.as_u64());
 		vesting.admin_release(Sender(ADMIN));
-		let amount = vesting.get_total_amount();
-		assert_eq!(amount, 0);
+		let amount = vesting.get_released_amount();
```

Comment:
Change types in `VestingConfig` from `U512` to `u64`.

Reply:
Changing the type seems to break more things, the fix starts affecting the runtime calls. It seems far more convenient to work with U512 and convert to u64 when needed